### PR TITLE
[docs] added info for clientSecret Azure layout

### PR DESCRIPTION
**Pull request by kirill-kashin**
## Description

Adding information about clientSecret issued for 1 year to Azure layout.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

```changes
section: docs
summary: Add info about using client secrets in Azure.
type: fix
impact_level: low
```

### CODE REVIEW

File: candi/cloud-providers/azure/openapi/cluster_configuration.yaml [hunk 1]
```diff
@@ -365,7 +365,7 @@ apiVersions:
             type: string
           clientSecret:
             description: |
-              The client's secret.
+              The client's secret. Issued by default: for 1 year.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/cluster_configuration.yaml:368]:
```suggestion
              The client's secret. Issued by default for one year.
```

File: candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml [hunk 2]
```diff
@@ -208,7 +208,7 @@ apiVersions:
               Идентификатор клиента.
           clientSecret:
             description: |
-              Secret клиента.
+              Secret клиента. Выдается по умолчанию: на 1 год.
```
Comment (z9r5) [inline candi/cloud-providers/azure/openapi/doc-ru-cluster_configuration.yaml:211]:
```suggestion
              Secret клиента. Выдается по умолчанию на один год.
```

File: candi/cloud-providers/azure/docs/ENVIRONMENT_RU.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ description: "Настройка Azure для работы облачного п
    az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "DeckhouseCANDI"
    ```
 
+   > На этом этапе выдается [clientSecret](https://deckhouse.ru/documentation/v1/modules/030-cloud-provider-azure/cluster_configuration.html#azureclusterconfiguration-provider-clientsecret) по умолчанию на 1 год. Официальная документация для [обновления сертификата](https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-app-service-certificate?tabs=portal#renew-an-app-service-certificate)
```
Comment (z9r5) [inline candi/cloud-providers/azure/docs/ENVIRONMENT_RU.md:23]:
```suggestion
   > На этом этапе выдается [clientSecret](cluster_configuration.html#azureclusterconfiguration-provider-clientsecret) по умолчанию на один год. Подробнее об обновления сертификата читайте в [официальной документации](https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-app-service-certificate?tabs=portal#renew-an-app-service-certificate).
```

File: candi/cloud-providers/azure/docs/ENVIRONMENT.md [hunk 4]
```diff
@@ -20,6 +20,8 @@ You have to create a service account with Microsoft Azure so that Deckhouse can
    az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "DeckhouseCANDI"
    ```
 
+   > This step issues a default [clientSecret](https://deckhouse.io/documentation/v1/modules/030-cloud-provider-azure/cluster_configuration.html#azureclusterconfiguration-provider-clientsecret): for 1 year. Official documentation for [certificate renewal](https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-app-service-certificate?tabs=portal#renew-an-app-service-certificate)
```
Comment (z9r5) [inline candi/cloud-providers/azure/docs/ENVIRONMENT.md:23]:
```suggestion
   > This step issues a default [clientSecret](cluster_configuration.html#azureclusterconfiguration-provider-clientsecret): for 1 year. Read more about certificate renewal in the [official documentation](https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-app-service-certificate?tabs=portal#renew-an-app-service-certificate).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7678](https://github.com/deckhouse/deckhouse/pull/7678) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7679](https://github.com/deckhouse/deckhouse/pull/7679) to the branch [release-1.57](https://github.com/deckhouse/deckhouse/tree/release-1.57) successful!

**Pull request merged**